modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -9,7 +9,7 @@ class BloopTests extends munit.FunSuite {
   val dummyInputs = TestInputs(
     Seq(
       os.rel / "Test.scala" ->
-        """// using scala 2.13
+        """// @using scala "2.13"
```

Comment:
I thought we were not supporting `// @using` (only `// using` or `@using`)

Reply:
Yes, it's changed in further commits